### ISSUE TITLE
Blog: categories "highlight" state

### ISSUE THREAD

**Issue by chrismplus** [mplusmuseum/mplusmuseum-stories#12]
**Issue:** Can we find a clearer design solution for the 'active' state re: categories instead of using the bullet point? There's also a spacing difference between the "All" category and the dot, and the other categories and the dot.

**Comment by chrismplus** [comment]
@rdunk hold on this one may no longer be relevant after Phase 2.

**Comment by chrismplus** [comment]
No longer relevant with new blog design. Closing issue.
